Darkly/LDClient.m

```diff
@@ -53,6 +55,13 @@ - (BOOL)start:(LDConfigBuilder *)inputConfigBuilder userBuilder:(LDUserBuilder *
                 LDClientManager *clientManager = [LDClientManager sharedInstance];
                 [clientManager syncWithServerForConfig];
                 [clientManager startPolling];
+                
+                EventSource *source = [EventSource eventSourceWithURL:[NSURL URLWithString:kStreamUrl] apiKey:ldConfig.apiKey];
```

Comment:
Currently, this will keep listening to events even in background. In the next few commits I'll implement disconnecting the stream, and add hard polling after every 5 minutes.